command/volume_status.go

```diff
@@ -191,6 +192,10 @@ func (c *VolumeStatusCommand) Run(args []string) int {
 			// for read, we only want to show whichever has results
 			hostErr := c.hostVolumeStatus(client, id, nodeID, nodePool, opts)
 			if hostErr != nil {
+				if !errors.Is(hostErr, hostVolumeListError) {
+					c.Ui.Error(hostErr.Error())
+					return 1 // we found a host volume but had some other error
```

Comment:
I love love the `%w`s everywhere, and functionally it looks great, but the word choice confuses me.

1. if there was an error
2. and it was not a "list host volumes" error
3. output the error and exit 1

it's # 2 that trips me up -- this "list volumes" error is actually a prefix search error, right? actual `hostVolumeList()` only happens if `id == ""` (and similar for csi)

Reply:
The error text is "Error listing host volumes" so that's where the name came from (the text matches everywhere else we're doing a prefix match). It could be a prefix search error. But it could also be any lower-level error like a ACL check failure or a connectivity failure or a "no such leader" error.

Reply:
alrighty, fair enough. I'm happy to approve after the tests are happy :+1: